app/views/search/index.html.erb

```diff
@@ -0,0 +1,76 @@
+<div class="l-header-library -search">
+  <div class="overlay"></div>
+  <%= render 'shared/menu' %>
+  <div class="row">
+    <div class="small-12 columns">
+      <div class="title-block">
+        <h1 class="text -title-library">Search</h1>
+        <%= render 'shared/share' %>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="l-header-library-separator"></div>
+
```

Comment:
I think it would be useful to have a summary of results at the top, but we might need to wait for designs from Juan Carlos. In the meanwhile having a "No results for your search" would be good to show.